.github/workflows/integration.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         run: docker-compose -f docker-compose-avro.yml pull --ignore-pull-failures
 
       - name: TiKV version
-        run: docker run pingcap/tikv:release-4.0-nightly -V
+        run: docker run pingcap/tikv:nightly -V
```

Comment:
Perhaps a conflict with https://github.com/pingcap/ticdc/issues/2442? We need to correspond with the branch.

Reply:
Yes, we need to change versions in cherry-picks.

Reply:
Only the actions in the master branch take effect, so we might need to add it now if we really want to run these tests in the release branch. But it looks like you don't have it enabled in the release branch yet.